x-pack/plugins/fleet/server/routes/package_policy/handlers.ts

```diff
@@ -371,7 +371,7 @@ export const updatePackagePolicyHandler: FleetRequestHandler<
         ...body,
         name: body.name ?? packagePolicy.name,
         description: body.description ?? packagePolicy.description,
-        namespace: body.namespace ?? packagePolicy.namespace,
+        namespace: body?.namespace ?? packagePolicy?.namespace,
```

Comment:
nit: does the `?` is needed here could `body` be undefined?

Reply:
Good catch, fixing it!